code/datums/components/ghost_direct_control.dm

```diff
@@ -143,12 +143,13 @@
 	if (extra_control_checks && !extra_control_checks.Invoke(harbinger))
 		return
 	harbinger.log_message("took control of [new_body].", LOG_GAME)
-	new_body.key = harbinger.key
-	to_chat(new_body, span_boldnotice(assumed_control_message))
-	after_assumed_control?.Invoke(harbinger)
+	harbinger.mind.transfer_to(new_body)
```

Comment:
Observers don't have minds, so this runtimes for its most common use case. Why was this change needed compared to setting the key?

Reply:
well they should

Reply:
yeah this is dumb as sit